### PR TITLE
Homework to lesson 43

### PR DESCRIPTION
**Pull request by RuslanTretyak**

### CODE REVIEW

File: src/main/java/application/util/PostgreDriverManager.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+@Component
+public class PostgreDriverManager {
+    @Value("${postgreDriverManager.url}")
+    private String URL;
```
Comment (DHrytsuk) [inline src/main/java/application/util/PostgreDriverManager.java]: It should be in camel case if it's not a constant

File: src/main/java/application/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @RequestMapping("/get-user")
+    public String getUserInfo(@RequestParam int id, Model model) {
+        User user = null;
+        try {
+            user = this.userService.getUserInfo(id);
+        } catch (SQLException e) {
```
Comment (DHrytsuk) [inline src/main/java/application/controller/UserController.java:29]: We should handle SQLException in service layer

File: src/main/java/application/controller/UserController.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+
+    @RequestMapping(value = "/create")
+    public ModelAndView showCreateUserForm() {
+        return new ModelAndView("create_user", "user", new User());
+    }
+
+    @RequestMapping(value = "/create", method = RequestMethod.POST)
+    public ModelAndView createUser(@ModelAttribute User user, Model model) {
+        if (!user.getLogin().isEmpty() && !user.getName().isEmpty() && !user.getSurname().isEmpty()) {
```
Comment (DHrytsuk) [inline src/main/java/application/controller/UserController.java]: Move this check to a separate method

File: src/main/java/application/util/PostgreDriverManager.java (hunk 1 above)
Comment (RuslanTretyak) [reply src/main/java/application/util/PostgreDriverManager.java]: исправил

File: src/main/java/application/controller/UserController.java (hunk 3 above)
Comment (RuslanTretyak) [reply src/main/java/application/controller/UserController.java]: использовал @Valid

File: src/main/java/application/controller/UserController.java (hunk 2 above)
Comment (RuslanTretyak) [reply src/main/java/application/controller/UserController.java:29]: переместил

File: src/main/java/application/util/PostgreDriverManager.java (hunk 1 above)
Comment (DHrytsuk) [reply src/main/java/application/util/PostgreDriverManager.java]: What about others?)

Comment (RuslanTretyak) [reply src/main/java/application/util/PostgreDriverManager.java]: да, вот теперь исправил

**Review by DHrytsuk — APPROVED** [review]

**Pull request closed without merging**